sympy/solvers/ode/tests/test_systems.py

```diff
@@ -781,11 +951,166 @@ def test_linear_3eq_order1_type4_slow():
     dsolve(eq1)
 
 
+def test_linear_neq_order1_type2_big_test_cases():
+    i, r1, c1, r2, c2, t = symbols('i, r1, c1, r2, c2, t')
+    x1 = Function('x1')
+    x2 = Function('x2')
+    eq1 = r1*c1*Derivative(x1(t), t) + x1(t) - x2(t) - r1*i
+    eq2 = r2*c1*Derivative(x1(t), t) + r2*c2*Derivative(x2(t), t) + x2(t) - r2*i
+    eq = [eq1, eq2]
+    x_1 = c1 ** 2 * r1 ** 2 + 2 * c1 ** 2 * r1 * r2 + c1 ** 2 * r2 ** 2 - 2 * c1 * c2 * r1 * r2 + 2 * c1 * c2 * r2 ** 2\
+          + c2 ** 2 * r2 ** 2
+    x_2 = exp(-t / (2 * c2 * r2) - t / (2 * c2 * r1) - t / (2 * c1 * r1) - t * sqrt(x_1) / (2 * c1 * c2 * r1 * r2))
+    x_3 = exp(-t / (2 * c2 * r2) - t / (2 * c2 * r1) - t / (2 * c1 * r1) + t * sqrt(x_1) / (2 * c1 * c2 * r1 * r2))
+    x_4 = Integral(-i * r2 * exp(
+        t / (2 * c2 * r2) + t / (2 * c2 * r1) + t / (2 * c1 * r1) + t * sqrt(x_1) / (2 * c1 * c2 * r1 * r2)) / sqrt(
+        x_1), t)
+    x_5 = Integral(i * r2 * exp(
+        t / (2 * c2 * r2) + t / (2 * c2 * r1) + t / (2 * c1 * r1) - t * sqrt(x_1) / (2 * c1 * c2 * r1 * r2)) / sqrt(
+        x_1), t)
+    x_6 = 1 / (c1 * r1 + c1 * r2 - c2 * r2 + sqrt(x_1))
+    x_7 = 1 / (c1 * r1 + c1 * r2 - c2 * r2 - sqrt(x_1))
+    sol = [Eq(x1(t),
+              -2 * C1 * c2 * r2 * x_2 * x_6 - 2 * C2 * c2 * r2 * x_3 * x_7 - 2 * c2 * r2 * x_2 * x_4 * x_6 - 2 * c2 *
+              r2 * x_3 * x_5 * x_7),
+           Eq(x2(t), C1 * x_2 + C2 * x_3 + x_2 * x_4 + x_3 * x_5)]
+    assert dsolve(eq) == sol
+    assert checksysodesol(eq, sol) == (True, [0, 0])
+
+
+def _de_lorentz_solution():
+    m = Symbol("m", real=True)
+    q = Symbol("q", real=True)
+    t = Symbol("t", real=True)
+
+    e1, e2, e3 = symbols("e1:4", real=True)
+    b1, b2, b3 = symbols("b1:4", real=True)
+    v1, v2, v3 = symbols("v1:4", cls=Function, real=True)
+
+    eqs = [
+        -e1 * q + m * Derivative(v1(t), t) - q * (-b2 * v3(t) + b3 * v2(t)),
+        -e2 * q + m * Derivative(v2(t), t) - q * (b1 * v3(t) - b3 * v1(t)),
+        -e3 * q + m * Derivative(v3(t), t) - q * (-b1 * v2(t) + b2 * v1(t))
+    ]
+
+    return eqs, dsolve(eqs), [m, q, t, e1, e2, e3, b1, b2, b3, v1, v2, v3]
+
+
+# A very big solution is obtained for this
+# test case. To be simplified in future.
+def test_linear_new_order1_type2_de_lorentz():
+
+    eqs, dsolve_sol, symbols = _de_lorentz_solution()
+    m, q, t, e1, e2, e3, b1, b2, b3, v1, v2, v3 = symbols
+    x_1 = Integral(b1**3*b2*e2*q*exp(q*t*sqrt(-b1**2 - b2**2 - b3**2)/m)/(2*b1**3*b3*m +
```

Comment:
This long expression for `x_1` can be simplified further by running `printsol` on it.